invenio_requests/config.py

```diff
@@ -117,3 +117,9 @@
     "is_open": {"facet": facets.is_open, "ui": {"field": "is_open"}},
 }
 """Available facets defined for this module."""
+
+REQUESTS_ENABLE_REVIEWERS = False
```

Comment:
Following other naming conventions with similar feature flags I would rename it to
```suggestion
REQUESTS_REVIEWERS_ENABLED = False
```